assets/styles/app.css

```diff
@@ -3,3 +3,75 @@
 
 /* Encore can't find '~select2-bootstrap-theme' with @import */
 @import '../../node_modules/select2-bootstrap-theme/dist/select2-bootstrap.min.css';
+
+.page-header {
+	background-color: #f5f5f5;
+	margin: 0 0 25px;
+	padding: 24px 0 18px;
+}
+
+.container {
+	max-width: 1170px;
+}
+
+.page-header .container {
+	align-items: center;
+	display: flex;
+	/* Width of container + logo size and it's padding */
+	max-width: calc( 1170px + ( (50px + 20px) * 2) );
+	width: auto;
+}
+
+.logo {
+	float: left;
```

Comment:
The logo is already a flex item, so shouldn't need to float. I think instead it needs no top and bottom margin, and only a right margin (to keep it away from the title text). At the moment it doesn't quite line up:

![ocr logo alignment](https://user-images.githubusercontent.com/213655/124069108-41cace80-da6e-11eb-8691-5a8619cc0b67.png)

without float and with the margin fixed it's like:

![ocr logo alignment fixed](https://user-images.githubusercontent.com/213655/124069114-42fbfb80-da6e-11eb-85ea-80a22547d41e.png)

...oh, hang on, sorry: i'm totally wrong, the way you've got it matches the designs! oops. Well, anyway, it probably still doesn't need the float...

Reply:
Yeah I was going off of the designs. I've removed the `float: left`, though (which I got from WS Export).